map/util/lanelet2_map_preprocessor/src/merge_close_lines.cpp

```diff
@@ -47,21 +44,17 @@ bool loadLaneletMap(
   return true;
 }
 
-bool exists(std::unordered_set<lanelet::Id> & set, lanelet::Id element)
-{
-  return set.find(element) != set.end();
-}
-
-lanelet::LineStrings3d convertLineLayerToLineStrings(lanelet::LaneletMapPtr & lanelet_map_ptr)
+lanelet::LineStrings3d convert_line_layer_to_line_strings(
+  const lanelet::LaneletMapPtr & lanelet_map_ptr)
 {
   lanelet::LineStrings3d lines;
-  for (const lanelet::LineString3d & line : lanelet_map_ptr->lineStringLayer) {
-    lines.push_back(line);
-  }
+  std::copy(
+    lanelet_map_ptr->lineStringLayer.begin(), lanelet_map_ptr->lineStringLayer.end(),
+    std::back_inserter(lines));
   return lines;
 }
 
-lanelet::ConstPoint3d get3DPointFrom2DArcLength(
+lanelet::ConstPoint3d get3_d_point_from2_d_arc_length(
```

Comment:
This function name sounds strange to me.
I think  `get_3d_point_from_2d_arc_length` is better.

Reply:
Yes, you are right. This was done by clang-tidy which I overlooked it

Reply:
I fixed it at https://github.com/autowarefoundation/autoware.universe/commit/99843aa6db2dff34ec218258d2458c9789bd1b2e